mappings/net/minecraft/entity/mob/GhastEntity.mapping

```diff
@@ -1,6 +1,10 @@
 CLASS net/minecraft/unmapped/C_fdzgodmg net/minecraft/entity/mob/GhastEntity
 	FIELD f_paccnhak SHOOTING Lnet/minecraft/unmapped/C_rinmcaxy;
 	FIELD f_tututexk fireballStrength I
+	METHOD m_fekrfazo returnedToSender (Lnet/minecraft/unmapped/C_sbxfkpyv;)Z
```

Comment:
hm, maybe
```suggestion
	METHOD m_fekrfazo hasReturnedToSender (Lnet/minecraft/unmapped/C_sbxfkpyv;)Z
```
...?

Reply:
If this was to be changed, maybe `wasReturnedToSender` would be even better?

Reply:
I like that!

Reply:
```suggestion
	METHOD m_fekrfazo wasReturnedToSender (Lnet/minecraft/unmapped/C_sbxfkpyv;)Z
```
:p